### DISCUSSION TITLE
Firewall plugin

### DISCUSSION THREAD

**Discussion by JHarrison712** [nautobot/nautobot#4023] [General]
Does anyone have a list of what firewalls are combability for Nautobot plugins? We are a big shop of Sonicwalls & pfSense & Fortigates and didn't know if there is a guide on something like this.

**Comment by chadell** [comment]
Hi @JHarrison712 , there is a generic firewall modeling app, https://github.com/nautobot/nautobot-plugin-firewall-models, that should help to model the data from different vendors
